test/onnx/test_operators.py

```diff
@@ -175,6 +175,12 @@ def test_transpose(self):
         x = torch.tensor([[0.0, 1.0], [2.0, 3.0]], requires_grad=True)
         self.assertONNX(lambda x: x.transpose(0, 1).transpose(1, 0), x)
 
+    def test_scatter(self):
+        data = torch.zeros(3, 3)
+        updates = torch.randn(2, 3)
+        indices=torch.tensor([[1,0,2],[0,2,1],])
+        self.assertONNX(lambda x: x.scatter_(0, indices, updates), data)
```

Comment:
We should be supporting x.scatter(), not x.scatter_(). Note `scatter()` is defined here https://fburl.com/owk2hefy and will likely need to be moved to ATen

Reply:
Oh yes sure, I just need to change it to be x.scatter().